### PR TITLE
Order listing page

### PR DESCRIPTION
**Pull request by jaypal412**
## Motivation and Context

Order List page UI
## Describe your changes

Changes made in the order index page, list page. In this new UI we have added a checkbox for the selection of the order.

## Any further comments?

------

## Checklist

- [ ] I have read [CONTRIBUTING.md][contributing].
- [ ] My code follows the [style guidelines][contributing] of this project.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] My code does not generate any (new) [`credo`][credo] and compile-time warnings.
- [ ] I have updated the documentation wherever necessary.
- [ ] I have added tests to cover my changes.

[contributing]: https://github.com/aviabird/snitch/CONTRIBUTING.md
[credo]: https://github.com/rrrene/credo

------
Dear Gatekeeper,

Please make sure that the commits that will be merged or rebased into the base
branch are [formatted according to our standards][commit-style].

> The only additional requirement is that the the PR number must appear in the
> commit title in brackets: `(#XXX)`

[commit-style]: https://github.com/aviabird/snitch/blob/develop/CONTRIBUTING.md#styling

### CODE REVIEW

File: apps/admin_app/config/config.exs [hunk 1]
```diff
@@ -36,8 +36,8 @@ config :admin_app, AdminAppWeb.Mailer,
   adapter: Swoosh.Adapters.Sendgrid,
   api_key: System.get_env("SENDGRID_API_KEY")
 
-config :pdf_generator,
-  wkhtml_path: "apps/admin_app/assets/wkhtml/wkhtmltopdf"
+# config :pdf_generator,
+#   wkhtml_path: "apps/admin_app/assets/wkhtml/wkhtmltopdf"
```
Comment (ashish173) [inline apps/admin_app/config/config.exs]: why is this commented? @jaypal412

File: apps/admin_app/lib/admin_app_web/templates/order/order_list_card.html.eex [hunk 2]
```diff
@@ -1,37 +1,48 @@
...
+               <a class="options" href="#" role="button" id="dropdownMenuLink" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+               <img src="/images/dots.svg" alt="Default">
+               </a>
+               <div class="dropdown-menu dropdown-menu-right" aria-labelledby="dropdownMenuLink">
+               </div>
+            </div>
+         </div>
+      </div>
+   </div>
+</div>
```
Comment (ashish173) [inline apps/admin_app/lib/admin_app_web/templates/order/order_list_card.html.eex]: leave a new line at the end always? @jaypal412

File: apps/admin_app/lib/admin_app_web/templates/order/order_list_card.html.eex [hunk 3]
```diff
@@ -1,37 +1,48 @@
...
+                  Ordered <span><%= @order.inserted_at %></span>
+               </div>
+               <div class="std-shipping my-3"><span class="mr-1">Standard Shipping</span>  (US$ 0.00) </div>
+               <div class="address">
+                  <div class="ship"> Ship to </div>
+                  <%= render_address(@order.shipping_address) %>
+               </div>
+            </div>
+            <div class="dropdown float-right">
+               <a class="options" href="#" role="button" id="dropdownMenuLink" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
```
Comment (ashish173) [inline apps/admin_app/lib/admin_app_web/templates/order/order_list_card.html.eex]: `dropdownMenuLink ` naming should be like `dropdown-menu-link` @jaypal412

File: apps/admin_app/lib/admin_app_web/templates/order/order_list_card.html.eex [hunk 4]
```diff
@@ -1,37 +1,48 @@
...
+               <img src="/images/dots.svg" alt="Default">
+               </a>
+               <div class="dropdown-menu dropdown-menu-right" aria-labelledby="dropdownMenuLink">
```
Comment (ashish173) [inline apps/admin_app/lib/admin_app_web/templates/order/order_list_card.html.eex]: same here as above @jaypal412

File: apps/admin_app/assets/css/listing.scss [hunk 5]
```diff
@@ -106,11 +106,154 @@
...
+    section {
+      padding: 60px 0;
+    }
+
+    .nav-tabs .nav-item.show .nav-link,
+    .nav-tabs .nav-link.active {
+      color: $gray-800;
+      border-bottom: 4px solid $main-color !important;
+      font-size: 16px;
+
```
Comment (ashish173) [inline apps/admin_app/assets/css/listing.scss]: why is this line here?

File: apps/admin_app/lib/admin_app_web/templates/order/order_list_card.html.eex (hunk 3 above)
Comment (jaypal412) [reply apps/admin_app/lib/admin_app_web/templates/order/order_list_card.html.eex]: Done

File: apps/admin_app/lib/admin_app_web/templates/order/order_list_card.html.eex (hunk 4 above)
Comment (jaypal412) [reply apps/admin_app/lib/admin_app_web/templates/order/order_list_card.html.eex]: done

File: apps/admin_app/assets/css/listing.scss (hunk 5 above)
Comment (jaypal412) [reply apps/admin_app/assets/css/listing.scss]: Done
